docs/pages/core-concepts.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Teleport Core Concepts
+description: Learn the the key architectural components that make up Teleport.
+---
+
+Here are the core concepts describe a Teleport deployment. You will see these
+terms often when setting up and managing Teleport, so you should get familiar
+with them before following other pages in the documentation.
+
+## Teleport cluster
+
+The key concept of Teleport's architecture is the **cluster**. A Teleport
+cluster consists of the **Teleport Auth Service** and **Teleport Proxy Service**,
+plus several **Teleport Services** that manage traffic to resources within your
```

Comment:
```suggestion
plus one or more **Teleport Services** that manage traffic to resources within your
```

Reply:
Suggesting using "the", since our docs sometimes imply that a Teleport cluster consists only of the Auth/Proxy Service